src/components/Sing/ScoreSequencer.vue

```diff
@@ -710,6 +744,12 @@ const onMouseDown = (event: MouseEvent) => {
   if (!isSelfEventTarget(event)) {
     return;
   }
+  if (event.shiftKey) {
```

Comment:
ここの条件はこうかも？（onMouseUpの処理と合うように）
```suggestion
  if (event.shiftKey && event.button === 0) {
```